notebooks/k_means.livemd

```diff
@@ -28,6 +29,7 @@ The main purpose of this livebook is to introduce the **KMeans** clustering algo
 
 ```elixir
 alias VegaLite, as: Vl
+alias Vl.Data
```

Comment:
```suggestion
alias VegaLite.Data
```

for aliasing, unless I'm aliasing a really long module, I'd rather avoid using the alias instead of the fully qualified name for module prefixes

Reply:
Ok, that makes sense. I'll push changes.